docs/workflow.md

```diff
@@ -88,22 +71,40 @@ We don't impose any rules for how you should be committing code, just keep the f
+
 ### Pull Requests
 
 Pull Requests give us a chance to run our automated tests and review the code before it gets merged. This helps us make sure our code on `upstream/master` always compiles and is as bug-free as possible.
 
 The code-review process gives us a chance ask questions or suggest improvements regarding a proposed change, so that the code is of the highest possible quality before being merged. It is also a good opportunity for others on the team to see what changes are being made, even if they are not involved in the project.
 
 The Pull Request process usually looks like the following:
-
+ 
 1. Make sure all the changes you want to make are pushed to a branch on your fork of the repository
-2. From the main page of your fork of the Software repository, click on the "code" tab and then on the "branches" tab below.
-3. Find the branch you want to open a Pull Request with and click "New pull request"
-4. Make sure the target (base-fork) is the `UBC-Thunderbots/Software` repository with branch `master`
-5. Give your Pull Request a short but descriptive title (the title should reflect the changes)
-6. Fill out the pull request template. This includes things like a description of the changes, indicating which issues the Pull Request resolves, and indicating what testing has been done.
-7. Add reviewers. This should be anyone that worked with you on the changes or is working on something that will be affected by the changes. Add your team lead and a few other members. Around 5 reviewers is a good number, but use your best judgement. Remember, these reviews also help give other team members an idea of the changes that are being made even if they aren't working on them.
-8. Tag the Pull Request with any relevant tags
+2. Make sure you have [updated your branch](#updating-your-branch-and-resolving-conflicts) and [formatted your code](#formatting-your-code). This is to help make sure CI will pass.
+3. From the main page of your fork of the Software repository, click on the "code" tab and then on the "branches" tab below.
+4. Find the branch you want to open a Pull Request with and click "New pull request"
+5. Make sure the target (base-fork) is the `UBC-Thunderbots/Software` repository with branch `master`
+6. Give your Pull Request a short but descriptive title (the title should reflect the changes)
+7. Fill out the pull request template. This includes things like a description of the changes, indicating which issues the Pull Request resolves, and indicating what testing has been done.
+8. Add reviewers. This should be anyone that worked with you on the changes or is working on something that will be affected by the changes. Add your team lead and a few other members. Around 3-4 reviewers is a good number, but use your best judgement. Remember, these reviews also help give other team members an idea of the changes that are being made even if they aren't working on them.
```

Comment:
I would add that the Owners also have to review each change.

Reply:
done